### ISSUE TITLE
id override 

### ISSUE THREAD

**Issue by rraammiinn** [manatlan/htag#11]
isn't there any way for overriding id of elements ?

**Comment by manatlan** [comment]
no, it's not possible ... (as all htag mechanism is based on ids)

but ... it's possible to create something like that;

```
self <= Tag.div(   "<span id='myid'>hello</span>" )
```
and use the id later, but it's not a good practice. Cause it could complexify your app.
But for some special needs, it can be useful

**Comment by rraammiinn** [comment]
thanks .
